internal/controller/util.go

```diff
@@ -71,6 +71,5 @@ func enqueueOnJobCompletion(e event.UpdateEvent) bool {
 }
 
 func finVolumePVCName(backup *finv1.FinBackup) string {
```

Comment:
Don't you need to update the function name?
(e.g. `finPVCName`)

Reply:
I'd like to keep it as is since we refer this volume as "fin volume" in spec.